### ISSUE TITLE
Int64.toString() and MIN_INT64

### ISSUE THREAD

**Issue by azrafe7** [HaxeFoundation/haxe#4102]
```haxe
import haxe.Int64;

class Test {
    static function main() {
        var i64:Int64 = Int64.make(0x80000000, 0);
        trace(i64); // --9-2-2-3-3-7-20-3-6-8-5-4-7-7-5-80-8
    }
}
```

**Comment by Simn** [comment]
What in the world...

**Comment by azrafe7** [comment]
It seems there's also some (different kind of) problem with cpp (just updated from haxelib so 3.2.27, with latest haxe): http://hastebin.com/hofajihasa.tex

**Comment by danbrakeley** [comment]
Ha, I just hit this, and googling it brought me here. So this is still an issue in haxe 3.2.0, at least compiling to neko.

**Comment by fponticelli** [comment]
I hit that as well and thought that boundary values are:

public static var maxValue = haxe.Int64.make(0x7fffffff,0xffffffff);
public static var minValue = haxe.Int64.make(0x80000000,0x00000001);

Also I am pretty sure that `divMod` doesn't work on CPP.

Il giorno dom 23 ago 2015 alle ore 08:58 Dan Brakeley <
notifications@github.com> ha scritto:

**Comment by andyli** [comment]
Look like there is a pending PR: #4178

**Comment by andyli** [comment]
Fixed by #4178.
